db-client-java/src/main/java/com/eventstore/dbclient/AbstractPersistentSubscriptionSettingsBuilder.java

```diff
@@ -0,0 +1,141 @@
+package com.eventstore.dbclient;
+
+public class AbstractPersistentSubscriptionSettingsBuilder<T> {
```

Comment:
See related comments in [AbstractPersistentSubscriptionSettings](https://github.com/EventStore/EventStoreDB-Client-Java/pull/84/files#diff-3f22e62cf5819afff2ad131e5e7058c61029a3bc78c5490c78d1f057b8ebd9dbR3).